[2023-09-27T17:11:22.666-0400] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Server_Access_Log_Processing.extract manual__2023-09-27T21:06:06.163135+00:00 [queued]>
[2023-09-27T17:11:22.676-0400] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Server_Access_Log_Processing.extract manual__2023-09-27T21:06:06.163135+00:00 [queued]>
[2023-09-27T17:11:22.676-0400] {taskinstance.py:1359} INFO - Starting attempt 2 of 2
[2023-09-27T17:11:22.767-0400] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): extract> on 2023-09-27 21:06:06.163135+00:00
[2023-09-27T17:11:22.772-0400] {standard_task_runner.py:57} INFO - Started process 67508 to run task
[2023-09-27T17:11:22.775-0400] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'ETL_Server_Access_Log_Processing', 'extract', 'manual__2023-09-27T21:06:06.163135+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Server_Access_Log_Processing.py', '--cfg-path', '/tmp/tmpoz9og5p0']
[2023-09-27T17:11:22.776-0400] {standard_task_runner.py:85} INFO - Job 4: Subtask extract
[2023-09-27T17:11:22.896-0400] {task_command.py:415} INFO - Running <TaskInstance: ETL_Server_Access_Log_Processing.extract manual__2023-09-27T21:06:06.163135+00:00 [running]> on host erosennnin-TUF-Gaming-FX505DU-FX505DU
[2023-09-27T17:11:23.017-0400] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ramesh@somemail.com' AIRFLOW_CTX_DAG_OWNER='Ramesh Sannareddy' AIRFLOW_CTX_DAG_ID='ETL_Server_Access_Log_Processing' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2023-09-27T21:06:06.163135+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-27T21:06:06.163135+00:00'
[2023-09-27T17:11:23.018-0400] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-27T17:11:23.018-0400] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cut -f1,4 -d"#" web-server-access-log.txt > /home/erosennnin/Programing/Apache Airflow/dags/extracted.txt']
[2023-09-27T17:11:23.026-0400] {subprocess.py:86} INFO - Output:
[2023-09-27T17:11:23.027-0400] {subprocess.py:93} INFO - cut: web-server-access-log.txt: No such file or directory
[2023-09-27T17:11:23.028-0400] {subprocess.py:93} INFO - cut: Airflow/dags/extracted.txt: No such file or directory
[2023-09-27T17:11:23.028-0400] {subprocess.py:97} INFO - Command exited with return code 1
[2023-09-27T17:11:23.036-0400] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/erosennnin/anaconda3/envs/coursera/lib/python3.11/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-09-27T17:11:23.040-0400] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=ETL_Server_Access_Log_Processing, task_id=extract, execution_date=20230927T210606, start_date=20230927T211122, end_date=20230927T211123
[2023-09-27T17:11:23.144-0400] {standard_task_runner.py:104} ERROR - Failed to execute job 4 for task extract (Bash command failed. The command returned a non-zero exit code 1.; 67508)
[2023-09-27T17:11:23.186-0400] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-27T17:11:23.216-0400] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
